network/api/v1/api.go

```diff
@@ -53,6 +55,25 @@ func (a *Wrapper) Routes(router core.EchoRouter) {
 			return audit.StrictMiddleware(f, network.ModuleName, operationID)
 		},
 	}))
+
+	const diagnosticsBasePath = "/status/diagnostics/network"
+	for mapping, provider := range a.Service.MappedDiagnostics() {
+		fullPath := diagnosticsBasePath + "/" + mapping
+		router.GET(fullPath, func(c echo.Context) error {
+			results := provider()
+			var output interface{}
+			if len(results) == 1 {
+				// If only 1 entry, flatten
+				output = results[0]
+			} else {
+				output = results
+			}
+			buf := new(bytes.Buffer)
+			_ = yaml.NewEncoder(buf).Encode(output)
```

Comment:
`application/yaml` results in file download. I prefer the default to be json, same as diagnostics

Reply:
nvm. content-type is `text/plain`